weblate/templates/paginator.html

```diff
@@ -1,95 +1,29 @@
-                 aria-label="{{ page_obj.paginator.per_page }}" />
-          <div class="input-group">
-            <input {% if paginator_form %}form="{{ paginator_form }}"{% endif %}
-                   type="number"
-                   min="1"
-                   max="{{ page_obj.paginator.num_pages }}"
-                   name="page"
-                   class="form-control"
-                   value="{{ page_obj.number }}"
-                   aria-label="{% trans "Jump to" %}"
-                   id="position-input-editable-input">
-            <span class="input-group-addon">
-              {% comment %}Translators: This is partial position indicator shown when editing position{% endcomment %}
-              {% blocktrans with page_obj.paginator.num_pages as total %}/ {{ total }}{% endblocktrans %}
-            </span>
-          </div>
-          {% if not paginator_form %}</form>{% endif %}
-      </a>
-    </li>
-    <li {% if not page_obj.has_next %}class="disabled"{% endif %}>
-      <a
-        {% if page_obj.has_next %}
-          rel="next"
-          {# djlint:off #}
-          href="?page={{ page_obj.next_page_number }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}"
-          {# djlint:on #}
-        {% endif %}
-        class="green">
-        {% if not LANGUAGE_BIDI %}
-          {% icon "page-next.svg" %}
-        {% else %}
-          {% icon "page-previous.svg" %}
-        {% endif %}
-      </a>
-    </li>
-    <li {% if page_obj.paginator.num_pages == page_obj.number %}class="disabled"{% endif %}>
-      <a href="?page={{ page_obj.paginator.num_pages }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}"
-         class="green">
-        {% if not LANGUAGE_BIDI %}
-          {% icon "page-last.svg" %}
-        {% else %}
-          {% icon "page-first.svg" %}
-        {% endif %}
-      </a>
-    </li>
+<nav class="pagination">
+  <ul>
+    {% if page_obj.has_previous %}
+      <li>
+        <a href="?{% querystring page=page_obj.previous_page_number %}">&laquo; Previous</a>
+      </li>
+    {% endif %}
+
+    {% for page_number in page_obj.paginator.page_range %}
+      {% if page_number == page_obj.number %}
+        <li class="active">
+          <span>{{ page_number }}</span>
+        </li>
+      {% else %}
+        <li>
+          <a href="?{% querystring page=page_number %}">{{ page_number }}</a>
+        </li>
+      {% endif %}
+    {% endfor %}
+
+    {% if page_obj.has_next %}
+      <li>
+        <a href="?{% querystring page=page_obj.next_page_number %}">Next &raquo;</a>
+      </li>
+    {% endif %}
```

Comment:
Can you please bring back the original markup?

* Page navigation is missing
* UI icons are no longer used
* RTL languages are no longer getting correct icons
* Strings are no longer localized